templates/forum/forum.html.twig

```diff
@@ -27,15 +27,17 @@
   {% spaceless %}
     <h1 class="page-heading forum-name-heading">
       <span class="forum-name-heading__prefix">/f/</span>
-      <strong class="forum-name-heading__name">{{ forum.name }}</strong>
+      <strong class="forum-name-heading__name">{{ forum.name }}{% if mod is defined %}/mod{% endif %}</strong>
     </h1>
   {% endspaceless %}
 
+  {% if mod is not defined %}
```

Comment:
why do you want to remove sorting in mod-mode?

Reply:
At first I couldn't convince the sortBy to work properly in this route. After more testing it looks like I've finally convinced it to work, so I'll update PR with that